Assets/Code/Interactables/Effects/Static Effects/Modifiers/AttachedModifier.cs

```diff
@@ -0,0 +1,22 @@
+﻿public abstract class AttachedModifier : AttachedStaticEffect {
+    public ModifierType modifierType;
+
+    // How much modifier is currently applied
+    public int amount;
+
+    // How much to change the modifier per turn (can be positive or negative)
+    // Can be used to make "temporary buffs" that last until EOT or statuses like DOT that decrement slightly over time
+    public int incrementAmount;
```

Comment:
just verifying - code to support this hasn't been implemented yet, right?

Reply:
Yeah, supporting code has not been implemented. This is really just the architecture for how the system will work